jdaviz/core/loaders/importers/spectrum2d/spectrum2d.py

```diff
@@ -30,7 +30,9 @@ def hdu_is_valid(hdu):
         True if the HDU is a valid light curve HDU, False otherwise.
     """
     return (len(getattr(hdu, 'shape', [])) == 2
-            and ('DISPAXIS' in hdu.header or hdu.header.get('CTYPE1', '') == 'WAVE'))
+            and ('DISPAXIS' in hdu.header
+                 or hdu.header.get('CTYPE1', '') == 'WAVE'
+                 or hdu.header.get('EXTNAME', '') == 'FLUX'))
```

Comment:
I don't love how general this is, but I don't see anything else in the header that would help to exclude other cases (images, etc).  Ultimately its up to the user to know what their data is if it can be parsed multiple different ways 🤷‍♂️